addons/pos_self_order/static/src/app/pages/landing_page/landing_page.js

```diff
@@ -110,7 +110,7 @@ export class LandingPage extends Component {
         if (
             this.selfOrder.hasPresets() &&
             !this.selfOrder.currentOrder.preset_id &&
-            this.selfOrder.ordering
+            this.selfOrder.config.self_ordering_mode != "nothing"
```

Comment:
```suggestion
```
I think you can simply delete the condition. If self_order_mode is nothing, you cannot access self at all.